ui/packages/ui-lib/src/labeled-content/labeled-content.tsx

```diff
@@ -13,64 +13,60 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Button, Stack, Typography } from '@mui/material';
+import { Stack, Typography } from '@mui/material';
 import { LabeledContentProps } from './labeled-content.types';
-import AddIcon from '@mui/icons-material/Add';
 
 const LabeledContent = ({
   label,
-  children,
+  caption,
+  children: verticalStackChildrenSlot,
   isRequired = false,
-  sx,
-  actionButtonProps,
+  verticalStackSx: verticalSx = {
+    '.MuiTextField-root': {},
+    '.MuiAutocomplete-root': {},
+  },
+  horizontalStackSx: horizontalStackSx,
+  horizontalStackChildrenSlot,
   ...typographyProps
 }: LabeledContentProps) => {
-  const { dataTestId, buttonText, ...buttonProps } = actionButtonProps || {};
+  const {
+    // @ts-expect-error
+    '.MuiTextField-root': textFieldRootSx,
+    // @ts-expect-error
+    '.MuiAutocomplete-root': autocompleteRootSx,
+    ...verticalStackSx
+  } = verticalSx;
   return (
     <Stack
       sx={{
         '.MuiTextField-root': {
-          mt: actionButtonProps ? 0 : 1.5,
+          mt: 1.5,
+          ...textFieldRootSx,
         },
         '.MuiAutocomplete-root': {
           mt: 1.5,
+          ...autocompleteRootSx,
         },
         mt: 2,
-        ...sx,
+        ...verticalStackSx,
```

Comment:
Hey! Can you help with understanding why we need verticalStack/horizontalStack spreading instead of common sx? Because we need both in one time and it's just a naming?

Reply:
Yes. At this point we have two stacks, so just one "sx" wouldn't be clarifying regarding which one we're targetting